File: javascript/resist.js
var aR = process.argv;
var S = 0, R = 0;
for (var i = 2; i < aR.length; i++){
    S += 1/aR[i]; 
}
R = 1.0/S;
console.log("Общее сопротивление равно: " + Math.round(R*100)/100 + " Ом");

File: kotlin/resist.kt
import kotlin.math.round

fun main(args: Array<String>){
    var S = 0.0
    for (r in args) {
        S += 1/r.toFloat()
    }
    var R = 1.0 / S
    println("Общее сопроотивление равно: " + "%.2f".format(R).toString() + " Ом")
    
}
